DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/layout/ht-top-nav.directive.js

```diff
@@ -18,14 +18,15 @@
             templateUrl: 'FrontEnd/src/client/app/layout/ht-top-nav.html'
         };
 
-        TopNavController.$inject = ['i18n', '$state', 'routerHelper'];
+        TopNavController.$inject = ['userService', 'i18n', '$state', 'routerHelper'];
         /* @ngInject */
-        function TopNavController(i18n, $state, routerHelper) {
+        function TopNavController(userService, i18n, $state, routerHelper) {
             var vm = this;
 
             // add i18n for localization
             vm.i18n = i18n;
             var states = routerHelper.getStates();
+            vm.user = userService;
```

Comment:
```html
<div ng-controller="ShellController as vmShell">
    <header class="clearfix">
        <ht-top-nav navline="vm.navline"></ht-top-nav>
```

And I've found some mistake in Shell.html. It is not your code, but nevertheless! we need to explore why there is `vm.navline` instead of `vmShell.navline`